test/tests/kernels/fv_burger/fv_burger.i

```diff
@@ -0,0 +1,63 @@
+[Mesh]
+  [./gen_mesh]
+    type = GeneratedMeshGenerator
+    dim = 1
+    xmin = 0
+    xmax = 10
+    nx = 50
+  [../]
+[]
+
+[Problem]
+  kernel_coverage_check = off
+[]
+
+[Variables]
+  [./v]
+    family = MONOMIAL
+    order = CONSTANT
+    fv = true
+  [../]
+[]
+
+[ICs]
+  [./v_ic]
+    type = FunctionIC
+    variable = v
+    function = 'if (x > 2 & x < 3, 0.5, 0)'
+  [../]
+[]
+
+[FVKernels]
+  [./burger]
```

Comment:
It would be great if we can keep using the regular block `[Kernels]` for FV kernels as well

Reply:
DGKernels have their own block.  Do you know if registering disparate classes with the same system base causes problems - would it work?

Reply:
My preference here is not strong, but I like the existence of `FVKernel` block. Finite volumes and finite elements are fundamentally different concepts so I think some separation is good for users